addon/components/select-2.js

```diff
@@ -1,4 +1,5 @@
 import Ember from "ember";
+import DS from "ember-data";
```

Comment:
I am not sure what this import will mean for users without ember data. Will there be an error? Or will the import be covered by https://github.com/ember-cli/ember-cli-shims?